man/docker-run.1.md

```diff
@@ -621,7 +621,7 @@ incompatible with any restart policy other than `none`.
    For the `overlay2` storage driver, the size option is only available if the backing fs is `xfs` and mounted with the `pquota` mount option.
    Under these conditions, user can pass any size less than the backing fs size.
 
-**--stop-signal**=*SIGTERM*
+**--stop-signal**=""
   Signal to stop a container. Default is SIGTERM.
```

Comment:
Should this maybe note the fact that the default can be defined by the image or something? (`STOPSIGNAL` in `Dockerfile`)

```suggestion
  Signal to stop a container. Default is defined by `STOPSIGNAL` in the image or `SIGTERM` if not defined.
```

Reply:
Ah, yes, makes sense to add, let me add some of that; thanks!